### PR TITLE
Release 4.6.4

### PR DESCRIPTION
**Pull request by BrianBatchelder**
- Enable use of UAMessageCenterMessageViewController class.
- Enable use of initWithNibName:bundle: method in UAMessageCenterMessageViewController and UADefaultMessageCenterMessageViewController classes.

### CODE REVIEW

File: src/AirshipBindings.iOS/ApiDefinition.cs [hunk 1]
```diff
@@ -2151,7 +2156,7 @@ interface UANotificationContent
 		UIUserNotificationCategory AsUIUserNotificationCategory { get; }
 
 		// -(UNNotificationCategory * _Null_unspecified)asUNNotificationCategory __attribute__((availability(ios, introduced=10.0)));
-		[iOS (10, 0)]
+        [Introduced(PlatformName.iOS, 10, 0, message: "Introduced in iOS 10")]
```
Comment (BrianBatchelder) [inline src/AirshipBindings.iOS/ApiDefinition.cs]: This just fixes a deprecation warning.

**Comment by rlepinski** [comment]
👍

**Comment by crow** [comment]
🍔

**Comment by marc-scig** [comment]
🏭

**Pull request merged**